### PR TITLE
dApp: Adds UDC token to withdrawal screen

### LINKED ISSUE #2237: RDN tokens not shown on the token withdrawal screen on mainnet
## Thanks for filing a bug report :-)

Reported by @jjjseee

## Steps to Reproduce
1. Deposit RDN to the UDC
2. Withdraw tokens from the UDC
3. Wait 100 blocks
4. Check etherscan if the withdraw-transaction has been completed for the Raiden Account (in this case: https://etherscan.io/tx/0x9aa881340667f0d81c9e1b5c25e963c38b30e4be5b58aa500a42471957b95a66)

## Expected Result
- I see the RDN tokens on the Withdrawal Screen and can withdraw them to the main account.

## Actual Result
- I don't see the tokens on the withdrawal screen and cannot withdraw the tokens.

   grafik

### PR DESCRIPTION
**Pull request by taleldayekh**
Fixes #2237 

**Short description**
The utility token should now always be displayed on the Withdrawal screen if there is a balance.

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1. Deposit SVT and RDN.
2. Visit the Withdrawal screen, the tokens should be visible for withdrawal.

### CODE REVIEW

File: raiden-dapp/src/components/account/Withdrawal.vue [hunk 1]
```diff
@@ -135,18 +137,25 @@ import Spinner from '@/components/icons/Spinner.vue';
...
   balances: Token[] = [];
   loading: boolean = true;
   withdrawing: boolean = false;
   withdraw: Token | null = null;
 
   async mounted() {
     this.balances = await this.$raiden.getRaidenAccountBalances();
+
+    if (this.udcToken.balance.gt(Zero)) {
+      this.balances = uniq([...this.balances].concat(this.udcToken));
```
Comment (andrevmatos) [inline raiden-dapp/src/components/account/Withdrawal.vue:156]: `uniq` must be applied over an array of comparable objects. I'm not sure you are sure the **reference** of the object in the` udcToken` variable would be the same as whatever gets returned by `getRaidenAccountBalances`.
Therefore, you should use `uniqBy`, passing a function like `(token) => token.address` as second param, in order to make the resulting array unique per token addresses.

**Review by andrevmatos — COMMENTED** [review]
One issue only, should be good to go after that gets fixed.

**Review by andrevmatos — APPROVED** [review]
lgtm, ty

**Comment by taleldayekh** [comment]
> One issue only, should be good to go after that gets fixed.

Thanks @andrevmatos , I've implemented the changes now.

**Pull request merged**